src/recordlinker/schemas/pii.py

```diff
@@ -222,12 +238,7 @@ def model_construct(
         is this method will not parse and validate the data, thus should only be used
         when the data is already cleaned and validated.
         """
-        obj = super(PIIRecord, cls).model_construct(_fields_set=_fields_set, **values)
-        obj.address = [Address.model_construct(**a) for a in values.get("address", [])]
-        obj.name = [Name.model_construct(**n) for n in values.get("name", [])]
-        obj.telecom = [Telecom.model_construct(**t) for t in values.get("telecom", [])]
-        obj.identifiers = [Identifier.model_construct(**i) for i in values.get("identifiers", [])]
-
+        obj = cls(**values)
```

Comment:
@m-goggins I have a feeling it might be better to discuss this on a call.  The differences between `cls.model_construct()` and `cls()` come down to performance.  The former just builds the pydantic object, the latter builds and validates it.  The general idea is that we don't need to validate data in the database as it was already validated/normalized during ingestion.  I'm interested to learn more about what issues you found with this approach.

Reply:
reverted back per our conversation